### PR TITLE
feat(toggle): Adding BEM class naming

### PR DESCRIPTION
**Pull request by pixelastic**
Business as usual. Adding BEM classes to the toggle widget (using
refinementList internally)

### CODE REVIEW

File: example/index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <!--[if lte IE 9]>
   <script src="https://cdn.polyfill.io/v2/polyfill.min.js"></script>
   <![endif]-->
-  <!-- <link rel="stylesheet" href="./debug.css"> -->
+  <link rel="stylesheet" href="./debug.css">
```
Comment (bobylito) [inline example/index.html]: Do we want the debug activated by default?

File: widgets/toggle/__tests__/toggle-test.js [hunk 2]
```diff
@@ -23,13 +21,13 @@ describe('toggle()', () => {
     it('throws when no facetName', () => {
       expect(() => {
         toggle({container: document.createElement('div')});
-      }).toThrow(usage);
+      }).toThrow(/Usage: /);
```
Comment (bobylito) [inline widgets/toggle/__tests__/toggle-test.js]: Nice to have regex for testing.

**Comment by bobylito** [comment]
LGTM but I'm not super fan of PR with changes that are not related to the subject (here changes in tests, the part with the regex)

**Comment by pixelastic** [comment]
The test was really too tied to the code, and the usage message wasn't even correct. I felt like opening a PR for that was too nitpicky, but why not.

**Comment by vvo** [comment]
> The test was really too tied to the code

But now the test is less specific, I mean the test could pass while the usage would not be up to date or completely broken.

Yes it's tied to the implementation but that's the goal no?

**Comment by pixelastic** [comment]
> But now the test is less specific, I mean the test could pass while the usage would not be up to date or completely broken.

That was actually the case :). The usage message was out of sync with the jsdoc that itself was out of sync with the code. Still, we had a green test.

With the regexp the test is less specific (we only test that we fire a usage warning in case of bad usage, without checking the usage message), but at least this will not give us a false sense of security.

I don't think testing for a full string equality here will help us, this is just a copy-paste magnet and in the end the correctness of the usage message is not tested, we just test that this is the same string in the code and the tests.

**Comment by bobylito** [comment]
You are right @pixelastic let's keep things simple (and sorry I like to rant too much it seems ;P)

**Comment by pixelastic** [comment]
No problem @bobylito, this was an interesting concern

**Pull request merged**